js/todoapp.js

```diff
@@ -0,0 +1,171 @@
+const demoTodo = [
+  {
+    name: "Follow Jijuzo on GitHub",
+    id: 1,
+    completed: false,
```

Comment:
You are using 2 approaches to differ your todos. You have `tasks` data structure that has `completed` flag. But at the same you use html attributes to differ "completed" todos from all todos. Pick the 1 solution (both are valid)

By the way, if you decide to stay with the attributes way, add a prefix `data-` to all of your custom attributes

`completed` attribute is not a valid attribute. If you want to add some custom attribute, consider using HTML data attributes: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes